cfme/infrastructure/config_management.py

```diff
@@ -520,3 +440,142 @@ def __init__(self, name=None, url=None, ssl=None, credentials=None, key=None):
+        nav._nav_to_fn('Configuration', 'Configuration Management')(None)
+
+    def resetter(self):
+        accordion.tree('Providers',
+                       version.pick({version.LOWEST: 'All Red Hat Satellite Providers',
+                                     '5.6': 'All Configuration Manager Providers'}))
+
+        tb.select('Grid View')
+
+    def am_i_here(self):
+        return summary_title() == 'All Configuration Management Providers'
+
+
+@navigator.register(ConfigManager, 'Add')
+class Add(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        cfg_btn('Add a new Provider')
+
+
+@navigator.register(ConfigManager, 'Edit')
+class Edit(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Edit Selected item')
+
+
+@navigator.register(ConfigManager, 'RefreshRelationships')
+class RefreshRelationships(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Refresh Relationships and Power states', invokes_alert=True)
+
+
+@navigator.register(ConfigManager, 'Remove')
+class Remove(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Remove selected items from the VMDB', invokes_alert=True)
+
+
+@navigator.register(ConfigManager, 'Details')
+class Details(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.click(Quadicon('{} Configuration Manager'.format(self.obj.name), None))
+
+    def am_i_here(self):
+        return summary_title() in ('Configuration Profiles under Red Hat Satellite '
+                                   'Provider "{} Configuration Manager"'.format(self.obj.name),
+                                   'Inventory Groups under Ansible Tower Provider'
+                                   ' "{} Configuration Manager"'.format(self.obj.name))
+
+
+@navigator.register(ConfigManager, 'EditFromDetails')
+class EditFromDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        cfg_btn('Edit this Provider')
+
+
+@navigator.register(ConfigManager, 'RefreshRelationshipsFromDetails')
+class RefreshRelationshipsFromDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        cfg_btn('Refresh Relationships and Power states', invokes_alert=True)
+
+
+# todo: not sure whether this works or not. it seems it wasn't used for a long time
+@navigator.register(ConfigProfile, 'Description')
+class Description(CFMENavigateStep):
+
+    def step(self):
+        navigate_to(self.obj.manager, 'Details')
```

Comment:
You can check to see if the step is used anywhere, however if it isnt and we need this, it hsould now become

```python
prerequisite = NavigateToAttribute('manager', 'Details')
```